.github/workflows/package-tests.yml

```diff
@@ -64,8 +71,15 @@ jobs:
 
       - name: Citus package tests
         run: |
-          export PROJECT_VERSION="${{ github.event.inputs.name }}"
-          [ -z ${PROJECT_VERSION} ]&& export PROJECT_VERSION=10.2.1
+          export PROJECT_VERSION_FROM_PACKAGING="${{ needs.metadata.outputs.citus_version }}"
+          if [[ ! -z ${PROJECT_VERSION_FROM_PACKAGING} ]]
+          then
+            export PROJECT_VERSION=${PROJECT_VERSION_FROM_PACKAGING}
+          else
+            export PROJECT_VERSION="${{ github.event.inputs.prj_ver }}"
+          fi
+          echo "Citus Version: ${PROJECT_VERSION} "
+          [ -z ${PROJECT_VERSION} ] && export PROJECT_VERSION=10.2.1
```

Comment:
why do we need this ?

Reply:
I meant, at what circumstances "PROJECT_VERSION" could be empty ? I think this shouldn't happen ever.

Reply:
When the pipeline is triggered directly on push, project version is empty. in that case for the sake of testing I want the pipeline to be executed to make sure that test pipeline is intact